README.md

```diff
@@ -1,739 +1,83 @@
-      			be kept instead of deleted after extraction. This flag
-			is only used if the extract-data flag is set.
-```
 
-### Compatibility with PowerSimData
+# REISE.jl
+**REISE.jl** is the Renewable Energy Simulation Engine developed by [Breakthrough
+Energy Sciences](https://science.breakthroughenergy.org/) (BES) to solve DCOPF
+problems. It is written and Julia and can be used with the BES software ecosystem (see
+[PowerSimData]) or in a standalone
+mode (see [Zenodo](https://doi.org/10.5281/zenodo.4538590) for sample data).
 
-Within the Python code in this repo, there is some code to maintain
-compatibility with the `PowerSimData` framework.
 
-Both `call.py` and `extract_data.py` can be called using a positional
-argument that corresponds to a scenario id as generated by the
-`PowerSimData` framework. Using this invocation assumes the presence
-of the `PowerSimData` infrastructure including both a Scenario List
-Manager and Execute List Manager. This option is not intended for manual
-simulation runs.
+## Main Features
+Here are a few things that **REISE.jl** can do:
+* Formulate Production Cost Model (PCM) into an optimization problem that can be solved
+  by professional solvers compatible with [JuMP]
+* Decompose a model that couldn't be solved all at once in memory (due to high spatial
+  or temporal resolution or both) into a series of shorter-timeframe intervals that are
+  automatically run sequentially.
+* Handle storage and demand flexibility
+* Handle adaptive infeasible/suboptimal/numeric issues via the homogenous barrier
+  algorithm (when using the [Gurobi] solver) and involuntary load shedding
 
-Note also the different naming convention for various directories by
-`PowerSimData` as compared to the options for the Python scripts within
-this repository.
 
-## Docker
+## Where to get it
+For now, only the source code is available. Clone or Fork the code here on GitHub.
 
-The easiest way to setup this engine is within a Docker image.
 
-There is an included `Dockerfile` that can be used to build the Docker image. With the Docker daemon installed and running, navigate to the `REISE.jl` folder containing the `Dockerfile` and build the image:
+## Dependencies
+**REISE.jl** relies on several Julia packages. The list can be found in the
+***Project.toml*** file located at the root of this package.
 
-```bash
-docker build . -t reisejl
-```
+Note that it is possible to use [Gurobi] as a solver.
 
-To run the Docker image, you will need to mount two volumes; one containing the
-`Gurobi` license file and another containing the necessary input files for the
-engine. 
 
-```bash
-docker run -it -v /LOCAL/PATH/TO/GUROBI.LIC:/usr/share/gurobi_license -v /LOCAL/PATH/TO/DATA:/usr/share/data reisejl bash
-```
+## Installation
+There are two options, either install all the dependencies yourself or setup the engine
+within a Docker image. Detailed installation notes can be found [here](docs). You will
```

Comment:
I think we need square brackets to use the url reference, e.g. `[here][docs]`

Reply:
Thanks. By the way, the link will exist once we merge the `ben/reisejl` branch in the **docs** repository.

Reply:
Done